src/BloomExe/Book/XMatterHelper.cs

```diff
@@ -42,8 +45,13 @@ public XMatterHelper(HtmlDom bookDom, string nameOfXMatterPack, IFileLocator fil
 			}
 			catch (Exception error)
 			{
+				var errorTemplate = LocalizationManager.GetString("Errors.XMatterNotFound",
+					"This Book called for Front/Back Matter pack named '{0}', but Bloom couldn't find that on this computer. You can either install a BloomPack that will give you '{0}', or go to Settings:Book Making and change to another Front/Back Matter Pack.");
```

Comment:
Now this message is very specific to File Not Found, so I'd say it should definitely be limited to that exception.